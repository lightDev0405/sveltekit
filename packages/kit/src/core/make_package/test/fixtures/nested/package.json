{
	"name": "nested",
	"version": "1.0.0",
	"description": "package-nested-test"
}
