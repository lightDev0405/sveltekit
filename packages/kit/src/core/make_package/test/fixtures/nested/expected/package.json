{
	"name": "nested",
	"version": "1.0.0",
	"description": "package-nested-test",
	"type": "module",
	"exports": {
		"./package.json": "./package.json",
		"./index.js": "./index.js",
		"./internal/Test.svelte": "./internal/Test.svelte",
		"./Test.svelte": "./Test.svelte",
		".": "./index.js"
	}
}
